lib/tasks/pact.rake

```diff
@@ -17,7 +17,7 @@ unless Rails.env.production?
     end
   end
 
-  task default: "pact:verify"
+  # task default: "pact:verify"
```

Comment:
good point, I've included it in acceptance criteria of https://trello.com/c/qjoezykg/901-reinstate-the-pact-broker-2